node/beacon/src/beacon_chain.rs

```diff
@@ -26,34 +26,34 @@ impl BeaconBlockChain {
 
 #[cfg(test)]
 mod tests {
-    use std::sync::Arc;
+//    use std::sync::Arc;
```

Comment:
remove

Reply:
I'll keep for the reference and remove after I add finalized, pending block logic.